trace-mini-agent/src/stats_flusher.rs

```diff
@@ -0,0 +1,87 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2023-Present Datadog, Inc.
+
+use async_trait::async_trait;
+use log::{debug, error, info};
+use std::{sync::Arc, time};
+use tokio::sync::{mpsc::Receiver, Mutex};
+
+use datadog_trace_protobuf::pb;
+use datadog_trace_utils::stats_utils;
+
+use crate::config::Config;
+
+const STATS_FLUSH_INTERVAL_SEC: u64 = 3;
```

Comment:
I missed it in previous review. Small nit and suggestion for the future:

this could also be configurable / at least stored in Config struct

Reply:
done